.github/workflows/python.yml

```diff
@@ -0,0 +1,22 @@
+name: python
+
+on:
```

Comment:
Same comment as `check_content.yaml`.

Reply:
Yes, and same reply :-)

> I'm curious to understand what about having these checks turned on would prevent someone from pushing code in a non-working state?

Especially as this one is about of Python code is broken, and it's absolutely fine for someone to push code in an intermediate state where the tests are broken - but all the more reason for the tests to be running, so that it is known that the tests are failing...

Reply:
General experience about dealing with failing tests is to try and discover them as soon as possible... The longer it's left before discovering them, the more commits there are that will be possible candidates, and the more time-consuming it will be to fix.

If people commit little-and-often, and push frequently, I would expect the failing test to be discovered very soon after the breakage was created, which would make it much easier to spot the point the problem happened - and then either fix it, or ask for help fixing it....